undefined_syms.txt

```diff
@@ -310,6 +310,27 @@ D_06008604 = 0x06008604;
 D_06001420 = 0x06001420;
 D_06009A90 = 0x06009A90;
 
+// z_en_tk
+D_04051DB0 = 0x04051DB0;
+D_040521B0 = 0x040521B0;
+D_040525B0 = 0x040525B0;
+D_040529B0 = 0x040529B0;
+D_04052DB0 = 0x04052DB0;
+D_040531B0 = 0x040531B0;
+D_040535B0 = 0x040535B0;
+D_040539B0 = 0x040539B0;
+D_06001144 = 0x06001144;
+D_06001FA8 = 0x06001FA8;
+D_06002F84 = 0x06002F84;
+D_06003B40 = 0x06003B40;
+D_06004340 = 0x06004340;
+D_06004B40 = 0x06004B40;
+D_0600ACE0 = 0x0600ACE0;
+D_0600BC90 = 0x0600BC90;
+D_0600BCA0 = 0x0600BCA0;
+D_0600BE40 = 0x0600BE40;
+D_8002B5EC = 0x8002B5EC;
```

Comment:
I believe this is a leftover and isn't supposed to be there?

Reply:
Yeah, I'll remove it.